.github/workflows/docker.yml

```diff
@@ -0,0 +1,79 @@
+name: Create and publish a Docker images
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+    branches:
+      - "master"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    name: PHP ${{ matrix.php-version }}, Fixer ${{ github.ref == 'refs/heads/master' && 'edge' || github.ref_name }}
+    if: github.repository == 'PHP-CS-Fixer/PHP-CS-Fixer'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php-version: '7.4'
+            alpine-version: '3.16'
+          - php-version: '8.0'
+            alpine-version: '3.16'
+          - php-version: '8.1'
+            alpine-version: '3.18'
+          - php-version: '8.2'
+            alpine-version: '3.18'
+          - php-version: '8.3'
+            alpine-version: '3.18'
```

Comment:
question: do we need to specify it? is it possible to have it autodetected ?

Reply:
No, we can't autodetect it, it's a value for Docker's build arg (parameterised Alpine version in `Dockerfile`).